Java_100/binary-trees/binarytrees.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * The Computer Language Benchmarks Game
+ * http://benchmarksgame.alioth.debian.org/
+ *
+ * based on Jarkko Miettinen's Java program
+ * contributed by Tristan Dupont
+ * *reset*
+ * Modified by Hyuk-Je Kwon to run 100 times
```

Comment:
in the future no need to put your name here IMO, it will be in the commit history anyway. Don't bother removing it for this, it has no impact.